packages/e2e-tests/publish-packages.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-console */
+import * as childProcess from 'child_process';
+import * as glob from 'glob';
+import * as path from 'path';
+
+const repositoryRoot = path.resolve(__dirname, '../..');
+
+// Create tarballs
+childProcess.execSync('yarn build:npm', { encoding: 'utf8', cwd: repositoryRoot, stdio: 'inherit' });
+
+// Get absolute paths of all the packages we want to publish to the fake registry
+const packageTarballPaths = glob.sync('packages/*/sentry-*.tgz', {
+  cwd: repositoryRoot,
+  absolute: true,
+});
+
+// Publish built packages to the fake registry
+packageTarballPaths.forEach(tarballPath => {
+  // `--userconfig` flag needs to be before `publish`
+  childProcess.execSync(`npm --userconfig ${__dirname}/test-registry.npmrc publish ${tarballPath}`, {
```

Comment:
would be nice to just use craft here (https://github.com/getsentry/craft/blob/master/src/targets/npm.ts#L160), but doesn't support custom user configs so we can stick with this.